biblib/views/permission_view.py

```diff
@@ -285,6 +292,60 @@ def read_access(cls, service_uid, library_id):
 
         return False
 
+    @staticmethod
+    def format_permission_payload(payload_info=None):
+        """
+        Format the permission info into plain text and HTML payloads for the email
+
+        :param payload_info: dict; keys:
+                                    name: library name
+                                    permission_data: dict w/ permissions (keys) and Boolean values
+                                    email: email address of user receiving email
+        :return: payload_plain, payload_html
+        """
+
+        lib_name = payload_info.get('name', None)
+        permission_data = payload_info.get('permission_data', None)
+
+        if not permission_data:
+            current_app.logger.warning('Must pass permission data. Payload info: {0}'.format(payload_info))
+            return None
+
+        readable_permissions = {'read': 'read only',
+                                'write': 'read and write',
+                                'admin': 'admin',
+                                'owner': 'owner'}
+
+        payload_plain_info = []
+        payload_html_info = {}
+        for p, value in permission_data['permission'].iteritems():
+            readable_permission = readable_permissions.get(p, None)
+            if readable_permission:
+                tmp = u'Library: {0} \n    Permission: {1} \n    Have permission? {2} \n'.format(lib_name, readable_permission, value)
```

Comment:
I can see how it would be confusing, but on the flip side, if a user used to have the "read & write" permission, but now they have the "admin" permission, and the email only lists the "admin" permission, they may wonder if they still have the "read & write" permission as well. In reality, it doesn't actually matter, since we only respect the highest level permission, but that's harder to explain to a user than just listing everything that's changed. Unless you have another thought?